lib/order.rb

```diff
@@ -0,0 +1,59 @@
+require_relative "menu.rb"
+require_relative 'twilio'
+class Order
+  include Twilio
+  attr_reader :basket, :menu, :orders
+  def initialize(menu = Menu.new)
+    @basket = []
+    @menu = menu
+    @orders = []
+  end
+
+  def get_available_dishes
+    @menu.dishes.select{ |dish| dish.available == true}
+  end
+  
+  def add_to_basket(item, count = 1)
+    count.times do
+      @basket << {item.name.to_sym => item.price}
+    end
+  end
+  
+  def find_dish(item)
+    available_dishes = get_available_dishes
+    wanted_dish = available_dishes.select{ |dish| dish.name == item.to_sym }
+    raise "Error, dish not available" if wanted_dish.empty?
+    wanted_dish[0]
```

Comment:
Is there another method you can use on line 24 that will return a single item rather than an array?  If so, you would no longer need to do `wanted_dish[0]`, which is a bit misleading.